test/unit/ajax.js

```diff
@@ -1578,16 +1578,19 @@ module( "ajax", {
 		}
 	} );
 
-	ajaxTest( "#13240 - jQuery.ajax() - support non-RFC2616 methods", 1, {
-		url: "data/echoQuery.php",
-		method: "PATCH",
-		success: function() {
-			ok( true, "success" );
-		},
-		error: function() {
-			ok( false, "error" );
-		}
-	});
+	// BrowserStack PATCH support sometimes breaks so on TestSwarm run the test in IE8 only.
```

Comment:
Hm, so browserstack can't fix that issue huh? Would need to be careful here when we switch from testswarm